learn-express-build-strock-trading-platform/server030.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express');
+const app = express();
+
+app.listen(3000, function() {
+  console.log('Your app is listening on port 3000.');
+});
+
+// 030.  Let's start by creating a basic endpoint at /hi.  If users go to http://localhost:3000/hi, they will receive the response "Hi there trader!".  Your express app has an 'app.get()' method that you can use, this takes two arguments: the path (in our case '/hi') and a callback.  The callback will be structured like:
```

Comment:
This can be simplified and broken up a bit. One lesson could briefly introduce `app.get()` and have users pass it the path `/hi` as an argument. The next could be passing it the callback 

```js
function (request, response) {

});
```

And the last could be using `response.send()` to send a message back to the client.

After this, it would be good to go back and refactor the code from ES5 to ES6 like your later examples. You don't have to explain this in too much detail since users should have learned this in previous projects, you can just show a similar callback function using arrow functions. You could start with refactoring/shortening `app.get()`, then go back and refactor `app.listen()`.